packages/spl-token/src/utils/discovery.ts

```diff
@@ -0,0 +1,117 @@
+import fs, { Stats } from 'fs'
+import path from 'path'
+import { config } from '@aleph-indexer/core'
+import { DiscoveryFn, DiscoveryFnReturn } from '../types.js'
+
+export async function discoveryFn(): Promise<DiscoveryFnReturn> {
+  // @note: Get addresses from env vars
+  const mintsSet = new Set(
+    config.SPL_TOKEN_MINTS ? config.SPL_TOKEN_MINTS.split(',') : [],
+  )
+  const accountsSet = new Set(
+    config.SPL_TOKEN_ACCOUNTS ? config.SPL_TOKEN_ACCOUNTS.split(',') : [],
+  )
+
+  // @note: Get addresses from storage
+  /*for await (const { key: item } of discoveryLevelStorage.getAll()) {
```

Comment:
We need to keep this feature as many discovery methods are not deterministic and need to track old discovered accounts and mints